src/views/forms/ClosePositionForm.tsx

```diff
@@ -258,6 +258,8 @@ export const ClosePositionForm = ({
     </$InputsColumn>
   );
 
+  const isFullClose = percent === SIZE_PERCENT_OPTIONS[MAX_KEY] || currentSizeBN.eq(size ?? 0);
```

Comment:
How important is including the `percent` in this? I feel like we could make this a selector/calculator after ensuring `currentInput === 'closePosition'`, then we can utilize from `PlaceOrderButtonAndReceipt`